cmd/bpf2go/main.go

```diff
@@ -289,7 +288,7 @@ func (b2g *bpf2go) convert(tgt target, arches []string) (err error) {
 	}
 
 	depFileName := goFileName + ".d"
-	if err := ioutil.WriteFile(depFileName, depFile, 0666); err != nil {
+	if err := os.WriteFile(depFileName, depFile, 0600); err != nil {
```

Comment:
Changing the mode here isn't correct, IMO. You lose the ability to specify what resulting mode you want using `umask`. It'd also be better if this was a separate commit, since it changes behaviour vs. a "mechanical" rename.

Reply:
`0600` should not be here. it got here by mistake, will drop it.